MakeHistos/macros/MiniNTupliser.C

```diff
@@ -378,6 +378,7 @@ void MiniNTupliser( TString sample = "", TString in_dir = "", TString out_dir =
 
   evt_sel.muPair_mass_min = 105; // Require at least one Higgs candidate pair, default 60
   obj_sel.mu_pt_min       =  10; // Lower muon pT threshold for muons not from Higgs, default 20
+  obj_sel.mu_iso_max	  = 0.4;
```

Comment:
I think we should move to a cut on miniIso, as the tZq analysis did, and as Hamburg has shown to be optimal for the 2 muon categories.  I've moved to these cuts here:
https://github.com/UFLX2MuMu/H2MuAnalyzer/blob/master/MakeHistos/src/ObjectSelection.cc#L42